pennylane/plugins/default_qubit.py

```diff
@@ -284,7 +287,8 @@ class DefaultQubit(Device):
         'T': T,
         'CNOT': CNOT,
         'SWAP': SWAP,
-        'CSWAP':CSWAP,
+        'CSWAP': CSWAP,
+        'Toffoli': Toffoli,
```

Comment:
Other new gates are supported by decompositions?

Reply:
Yep.  Since the `U3` gate and `U2` gate are now 'supported' by default qubit via a decomposition into Rot and PhaseShift, I thought it would be overkill to add native support (and also additional maintenance).

In addition, the default.qubit gate tests require significant refactoring, as they still use loops and dynamic attribute loading. So adding gates to default.qubit is currently much more complicated than it should be.